.env.sample

```diff
@@ -24,10 +26,10 @@ REACT_APP_NETWORK=rpc
 REACT_APP_SERVER_URL=http://localhost:8000
 
 # Web3 stuff config
```

Comment:
Haha maybe we can split these options into "Web3 connection config" and "Web3 signing config"